src/js/0-global.js

```diff
@@ -138,6 +143,47 @@ function detectOS() {
   if(matchedOS){ return matchedOS; } else { return null; }
 }
 
+function toJson(response) {
+  while (typeof response === 'string') {
+    try {
+      response = JSON.parse(response)
+    } catch (e) {
+      return null
+    }
+  }
+  return response
+}
+
+// load latest_nightly.json/nightly.json/releases.json/latest_release.json files
+// This will first try to load from openjdk<X>-binaries repos and if that fails
+// try openjdk<X>-release, i.e will try the following:
+
+// https://github.com/AdoptOpenJDK/openjdk10-binaries/blob/master/latest_release.json
+// https://github.com/AdoptOpenJDK/openjdk10-releases/blob/master/latest_release.json
```

Comment:
This is only for 'release' level builds or 'nightly' level builds?

Reply:
No, those are just examples, in fact at the moment I have only pushed nightlys, so only nightlys will be found, but either should work